Makefile

```diff
@@ -3,6 +3,8 @@ PREFIX = /usr/local
 SYSCONFDIR = /etc
 # used for the default storage path
 SHAREDSTATEDIR = /var
+# used for the default manpage path
+MANPAGEDIR = $(PREFIX)/share/man
```

Comment:
I am unsure if the PREFIX is used correctly here

Reply:
Please name it `MANDIR` to match what autoconf uses.